_i18n/de/resources/moneropedia/mining.md

```diff
@@ -1,15 +1,15 @@
 ---
-terms: ["mining", "miner", "miners"]
-summary: "the process of cryptographically computing a mathematical proof for a block, containing a number of transactions, which is then added to the blockchain"
+terms: ["mining", "miner", "miners", "Mining", "Miner", "Minings", "Miners", "Minern"]
+summary: "Prozess des kryptografischen Errechnens eines mathematischen Nachweises eines Blocks, der eine Anzahl von Transaktionen enthält, welche dann zur Blockchain hinzugefügt werden"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-The process of cryptographically computing a mathematical proof for a block, containing a number of transactions, which is then added to the blockchain.
+Der Prozess des kryptografischen Errechnens eines mathematischen Nachweises eines Blocks, welcher eine Anzahl von Transaktionen enthält, die dann zur Blockchain hinzugefügt werden.
 
-Mining is the distributed process of confirming transactions on the public ledger of all transactions, aka @blockchain.  Monero nodes use the blockchain to distinguish legitimate transactions from attempts to re-spend coins that have already been spent elsewhere.
+Mining ist der distribuierte Prozess des Bestätigens von Transaktionen im öffentlichen, alle Transaktionen umfassenden Kassenbuch, bekannt als @Blockchain. Monero-Nodes nutzen die Blockchain, um rechtmäßige Transaktionen von denen zu unterscheiden, in welchen versucht wird, bereits zuvor ausgegebene Coins erneut zu senden.
 
-Monero is powered strictly by Proof of Work. It employs a mining algorithm that has the potential to be efficiently tasked to billions of existing devices (any modern x86 CPU and many GPUs). Monero uses a variant of CryptoNight Proof of Work (PoW) algorithm, which is designed for use in ordinary CPUs and GPUs.
+Monero wird streng angetrieben von Proof-of-Work. Es beschäftigt einen Mining-Algorithmus mit dem Potenzial, effizient mit Milliarden von bestehenden Geräten (jeder moderne x86-CPU und viele GPUs) beauftragt zu werden. Monero verwendet eine Variante des CryptoNight-Proof-of-Work-Algorithmus, der für den Gebrauch in üblichen CPUs und GPUs entwickelt worden ist.
 
-The smart mining feature allows transparent CPU mining on the user's computer, far from the de facto centralization of mining farms and pool mining, pursuing Satoshi Nakamoto's original vision of a true P2P currency.
+Diese intelligente Eigenschaft des Minings erlaubt transparentes CPU-Mining auf dem Rechner des Nutzers, weit weg von faktisch zentralisierten Mining-Farms und Mining-Pools, und strebt damit Satoshi Nakamotos ursprüngliche Vision einer wahrhaftigen P2P-Währung an.
```

Comment:
@erciccione

Reply:
@stmax82 I agree some sections could be improved and better wording could be used, but that's out of scope for this PR. The thing to do now is to localize the guides making sure they are as comprehensible as possible to readers, but keeping the original meaning and structure, If some of the English wording needs to be changed, we need to open and discuss on another PR.

> What is the "smart mining feature" this sentence is referring to? I guess it's "ASIC resistance"

It's not related to ASIC resistance. Smart mining is a feature that allow people to solo-mine using the CPU their computer is not utilizing at the moment.

> What does "transparent" in "transparent CPU mining" mean?

I guess the author means that you have full control of the mining process, unlike in a pool.

> "The smart mining feature allows CPU mining on the user's computer,..." should be "The mining algorithm goes to great lengths to only allow efficient CPU/GPU mining, while preventing the use of ASICs,...", no?

The two things are related, but the two sentences have completely different meaning.

> Pool mining doesn't lead to centralization, does it?

It does, because the hashrate is concentrated on the pool, which we have to trust.